marketplace_builder/pages/getting-started/notifications/creating-sms-notification.liquid

```diff
@@ -0,0 +1,90 @@
+---
+converter: markdown
+metadata:
+  title: Creating an SMS Notification
+slug: get-started/notifications/creating-sms-notification
+---
+
+This guide will help you create an SMS notification to send a text message to a newly signed up user using our Twilio integration. 
+
+## Requirements
+To follow the steps in this tutorial, you should understand the concept of notifications. You'll also use the sign up form you created in a previous tutorial. As you'll use our Twilio integration, you need to ensure that the `to` number is formatted according to Twilio's phone number format guideline. 
+
+* [Notifications](/get-started/notifications/notifications)
+* [Creating a Sign Up Form](/get-started/users/creating-sign-up-form)
+* [Twilio's phone number format guideline](https://support.twilio.com/hc/en-us/articles/223183008-Formatting-International-Phone-Numbers)
+
+
+## Steps 
+
+Creating an SMS notification is a three-step process:
+
+1. Extend sign up form with mobile number
+2. Create SMS notification 
+3. Add SMS notification to sign up form
+
+### Step 1: Extend sign up form with mobile number 
+
+As you haven't collected the mobile phone number in the sign up form yet, as a first step, extend the sign up form `form_configurations/developer/sign_up.liquid`:
+
+{% raw %}
+
+```liquid
+---
+configuration:
+  mobile_number:
```

Comment:
Since we are already showing some validation here, we can add:

```
phone_number: true
```

It provides couple basic checks to prevent you from sending ie. "HELLOWORLD" ;)

Example from live project:

```
        mobile_number:
          validation:
            presence: true
            phone_number: true
```